src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -12,6 +12,8 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
+    <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
```

Comment:
This will be broken after I merge my changes because now this reference will be to the 4.2 version of the assembly, but the implementation will still just get 4.1 causing a mismatch. To make this not break my code, could you try adding a target group on all of this P2P references? This will let you still reference the 4.1 version of the assembly after my change. This is more or less what I'm suggesting:
```xml
<ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj">
   <TargetGroup>netstandard1.5</TargetGroup>
</ProjectReference>
```
I believe that if you do that, it will work before and after my change.